### ISSUE TITLE
Google Fit to Fitbit?

### ISSUE THREAD

**Issue by Redth** [praveendath92/fitbit-googlefit#17]
How about syncing the other way? I'd like to add my google fit data to my Fitbit account...

**Comment by praveendath92** [comment]
@Redth Unfortunately Fitbit doesn't support writing steps or heart rate data.

**Comment by Redth** [comment]
@praveendath92 has this perhaps changed since you last looked? https://dev.fitbit.com/docs/activity/#activity-logging

Not seeing a way to create an entry for heart rate still but perhaps omitting that is OK.

**Comment by praveendath92** [comment]
I think that only supports manually logging activities and distance but not steps.

**Comment by praveendath92** [comment]
@Redth , I just came across [this](https://fitnessconnect.io/) so, I suppose it is possible to sync steps.

**Comment by Redth** [comment]
Yep I do believe it's possible.  Fitbit doesn't quite count API steps as regular steps (eg: they won't count towards challenges), but it would still be useful.

**Comment by praveendath92** [comment]
@Redth , Noted. I will update you here if I create something for the reverse sync - mostly in a different repo.

**Comment by Redth** [comment]
Thanks :)

**Comment by Melantrix** [comment]
Hi @praveendath92, i'm also very interested in syncing information (especially weight, but also activities) from google fit to fitbit, have you made any progress with this?

**Comment by praveendath92** [comment]
@Melantrix Sorry, a reverse sync is not in the plan for now.
